delirium/dns/fakednsserver.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, addr=DEFAULT_LISTEN_ADDR, port=DEFAULT_LISTEN_PORT, duration=
         self._addr = addr
         self._port = port
 
-        self._cache = database.create_cache(ip_range, duration, cache_type, cache_path)
+        self._cache = database.create_cache(ip_range, duration, cache_path)
```

Comment:
There's no longer a need for the `database.create_cache` helper function, go ahead and create a `CacheDatabase` object directly here

Reply:
This is also slated to be resolved as part of #24. The partial-implementation of db-only caching was only a side-effect of trying to eliminate the option out of the CLI during the CLI redesign.